system-addon/test/functional/mochitest/head.js

```diff
@@ -13,6 +19,13 @@ const ACTIVITY_STREAM_PREF = "browser.newtabpage.activity-stream.enabled";
 pushPrefs([ACTIVITY_STREAM_PREF, true]);
 gBrowser.removePreloadedBrowser();
 
+function clearHistoryAndBookmarks() { // eslint-disable-line no-unused-vars
+  return (async function() {
```

Comment:
Why have this return an async function call instead of just being async?

Reply:
I took this from our xpcshell tests https://searchfox.org/mozilla-central/source/toolkit/modules/tests/xpcshell/head.js#35. I'll update it.